packages/webapp/pages/account/subscription.tsx

```diff
@@ -23,37 +39,97 @@ const seo: NextSeoProps = {
   const router = useRouter();
-  const { isPlus, plusProvider } = usePlusSubscription();
+  const { openModal } = useLazyModal();
+  const { isPlus, plusProvider, logSubscriptionEvent } = usePlusSubscription();
 
   useEffect(() => {
     if (!isPlus) {
       router.push(plusUrl);
-      return;
-    }
-
-    if (plusProvider === SubscriptionProvider.Paddle) {
-      router.push(managePlusUrl);
-      return;
-    }
-
-    if (isIOSNative() && plusProvider === SubscriptionProvider.AppleStoreKit) {
-      postWebKitMessage(WebKitMessageHandlers.IAPSubscriptionManage, null);
-
-      // Send the user back to the previous page as the native handler will take over
-      router.back();
     }
-  }, [isPlus, plusProvider, router]);
+  }, [isPlus, router]);
 
   if (!router.isReady) {
     return null;
   }
 
   return (
-    <AccountPageContainer title="Manage plus">
-      <span className="text-text-tertiary typo-callout">
-        Your plus subscription is managed via App Store, to manage it please
-        visit the App Store
-      </span>
+    <AccountPageContainer title="Payment & Subscription">
+      <div className="flex flex-col gap-6">
+        <PlusUser
+          iconSize={IconSize.XSmall}
+          typographyType={TypographyType.Callout}
+        />
+
+        <div className="flex flex-col gap-1">
+          <Typography bold type={TypographyType.Body}>
+            You&apos;re already a Plus member!
+          </Typography>
+          <Typography
+            type={TypographyType.Callout}
+            color={TypographyColor.Tertiary}
+          >
+            Thanks for supporting daily.dev and unlocking our most powerful
+            experience.
+          </Typography>
+
+          {!isIOSNative() &&
+            plusProvider === SubscriptionProvider.AppleStoreKit && (
+              <Typography
+                type={TypographyType.Callout}
+                color={TypographyColor.Tertiary}
+                className="mt-2"
+              >
+                Your plus subscription is managed via App Store, to manage it
+                please visit the App Store
+              </Typography>
+            )}
+        </div>
+
+        <div className="flex gap-3">
+          <Button
+            tag="a"
+            size={ButtonSize.Small}
+            variant={ButtonVariant.Secondary}
+            href={managePlusUrl}
+            target="_blank"
+            disabled={
+              !isIOSNative() &&
+              plusProvider === SubscriptionProvider.AppleStoreKit
+            }
+            onClick={() => {
+              if (
+                isIOSNative() &&
+                plusProvider === SubscriptionProvider.AppleStoreKit
+              ) {
+                postWebKitMessage(
+                  WebKitMessageHandlers.IAPSubscriptionManage,
+                  null,
+                );
+              }
+
+              logSubscriptionEvent({
+                event_name: LogEvent.ManageSubscription,
```

Comment:
@idoshamun what do you want as `target_id` here? It's for the https://as-1098-layout.preview.app.daily.dev/account/subscription page

Reply:
account?

Reply:
👍

Reply:
Updated